app/services/wallet_service.py

```
@@ -0,0 +1,153 @@
+        self.currencies_repository = currencies_repository
+        self.gw2_client = gw2_client
+
+    async def get_wallet(self, account_uuid: UUID) -> list[WalletItemResponse]:
+        """
+        Get wallet data from GW2 API and sync with database in the background.
+        If API fails, fallback to database.
+        """
+        try:
+            wallet_data = await self._get_wallet_from_api()
+            # Sync with DB in background
+            asyncio.create_task(self._sync_wallet_to_db(wallet_data, account_uuid))
```

Ruff (0.14.2)

Store a reference to the return value of `asyncio.create_task`

(RUF006)